account_balance_reporting/models/account_balance_reporting_report.py

```diff
@@ -374,12 +374,14 @@ def refresh_values(self):
                                 ('date', '<=', report.previous_date_to)]
             # Exclude closing moves (if account_fiscal_year_closing installed)
             if 'closing_type' in self.env['account.move']._fields:
-                domain_current.append(
-                    ('move_id.closing_type', '!=', 'closing')
-                )
-                domain_previous.append(
-                    ('move_id.closing_type', '!=', 'closing')
-                )
+                domain_current.extend([
+                    ('move_id.closing_type', '!=', 'closing'),
+                    ('move_id.closing_type', '!=', 'loss_profit')
+                ])
+                domain_previous.extend([
+                    ('move_id.closing_type', '!=', 'closing'),
+                    ('move_id.closing_type', '!=', 'loss_profit')
+                ])
```

Comment:
Por qué no `('move_id.closing_type', 'not in', ('closing', 'loss_profit'))` ? es más lenta la query?
Lo pregunto porque no lo sé